src/Germany/Mews.Fiscalizations.Germany/V2/Model/Result/ErrorResult.cs

```diff
@@ -37,6 +37,11 @@ internal static ErrorResult MapException(JsonReaderException exception, string r
 
     private static FiskalyError MapError(Dto.FiskalyErrorResponse error)
     {
+        if (error is null)
+        {
+            return FiskalyError.InvalidResponse;
+        }
```

Comment:
This cannot be ever null based on its current usage, no? i believe it would crash on the deserialization and since there is a try/catch, it would use the MapException method and not this one

Reply:
Hey, I believe in this case the Json deserialization will just convert to a null object and not throw any exception so we could actually receive a null value